Crust/Mapper/MappingOperator.swift

```diff
@@ -30,7 +35,11 @@ public func <- <T: Mappable, U: Mapping, C: MappingContext where U.MappedObject
 // NOTE: Must supply two separate versions for optional and non-optional types or we'll have to continuously
 // guard against unsafe nil assignments.
 
-public func <- <T: JSONable, C: MappingContext where T == T.ConversionType>(inout field: T?, map:(key: JSONKeypath, context: C)) -> C {
+public func <- <T: JSONable, U: Transform, C: MappingContext where U.MappedObject == T, T == T.ConversionType, T: AnyMappable>(inout field: T?, map:(key: KeyExtensions<U>, context: C)) -> C {
+    return mapField(&field, map: map)
+}
+
+public func <- <T: JSONable, C: MappingContext where T == T.ConversionType >(inout field: T?, map:(key: JSONKeypath, context: C)) -> C {
```

Comment:
Seems like you append an additional space at `ConversionType >`. Delete please :)